lib/fog/libvirt/requests/compute/list_volumes.rb

```diff
@@ -31,7 +31,7 @@ def volume_to_attributes(vol)
             :format_type => format_type,
             :allocation  => bytes_to_gb(vol.info.allocation),
             :capacity    => bytes_to_gb(vol.info.capacity),
-          }
+          } rescue nil # If there are issues during stat of volume file
```

Comment:
Could you define an exception instead of rescue everything?
Otherwise this code will just swallow any other issues in that code.